lib/mpool/src/mpool_file.c

```diff
@@ -61,6 +61,10 @@ mpool_file_open(
     if (create)
         flags |= (O_CREAT | O_EXCL);
 
+    /* Drop ODIRECT if the mclass doesn't support direct-io */
+    if ((flags & O_DIRECT) && !mclass_directio(mc))
```

Comment:
Maybe name the function `mclass_is_direct_io_enabled()`?

Note that I split `directio` to `direct_io` based on the earlier parameter comments

Reply:
Kind of a verbose name, but gets the point across easily.

Reply:
Yeah, it is verbose for an internal function. I can do mclass_directio_get(mc).